From penberg@cs.helsinki.fi  Wed Jan 21 09:33:22 2009
From: Pekka J Enberg <penberg@cs.helsinki.fi>
Date: Thu, 8 Jan 2009 11:31:28 +0200 (EET)
Subject: Staging: w35und: remove useless macro from common.h
To: Greg Kroah-Hartman <greg@kroah.com>
Cc: Pavel Machek <pavel@suse.cz>, linux-kernel@vger.kernel.org
Message-ID: <Pine.LNX.4.64.0901081130530.15334@melkki.cs.Helsinki.FI>


From: Pekka Enberg <penberg@cs.helsinki.fi>

The DebugUsbdStatusInformation macro doesn't do anything useful so remove the
definition and the two users of it from driver code.

Signed-off-by: Pekka Enberg <penberg@cs.helsinki.fi>
Acked-by: Pavel Machek <pavel@suse.cz>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/winbond/common.h  |    3 ---
 drivers/staging/winbond/wb35reg.c |    1 -
 drivers/staging/winbond/wb35rx.c  |    1 -
 3 files changed, 5 deletions(-)

--- a/drivers/staging/winbond/common.h
+++ b/drivers/staging/winbond/common.h
@@ -4,9 +4,6 @@
 // This file contains the OS dependant definition and function.
 // Every OS has this file individual.
 //
-
-#define DebugUsbdStatusInformation( _A )
-
 #ifndef COMMON_DEF
 #define COMMON_DEF
 
--- a/drivers/staging/winbond/wb35reg.c
+++ b/drivers/staging/winbond/wb35reg.c
@@ -481,7 +481,6 @@ Wb35Reg_EP0VM_complete(struct urb *urb)
 		if (reg->EP0VM_status) {
 #ifdef _PE_REG_DUMP_
 			WBDEBUG(("EP0 IoCompleteRoutine return error\n"));
-			DebugUsbdStatusInformation( reg->EP0VM_status );
 #endif
 			reg->EP0vm_state = VM_STOP;
 			pHwData->SurpriseRemove = 1;
--- a/drivers/staging/winbond/wb35rx.c
+++ b/drivers/staging/winbond/wb35rx.c
@@ -195,7 +195,6 @@ static void Wb35Rx_Complete(struct urb *
 	if (pWb35Rx->EP3VM_status != 0) {
 		#ifdef _PE_USB_STATE_DUMP_
 		WBDEBUG(("EP3 IoCompleteRoutine return error\n"));
-		DebugUsbdStatusInformation( pWb35Rx->EP3VM_status );
 		#endif
 		pWb35Rx->EP3vm_state = VM_STOP;
 		goto error;
